ethcore/src/executive.rs

```diff
@@ -257,46 +267,45 @@ impl<'a> Executive<'a> {
 				}
 			}
 		} else {
-			// if destination is a contract, do normal message call
-			
-			// don't trace if it's DELEGATECALL or CALLCODE.
```

Comment:
There is a comment here: https://github.com/ethcore/parity/pull/903/files#diff-1562d2c35bff11a4bbbf1d2ba79c68d8R117